docker-core.el

```diff
@@ -128,7 +128,8 @@
    ("c" (lambda ()(plist-get docker-status-strings :containers)) docker-containers)
    ("i" (lambda ()(plist-get docker-status-strings :images))     docker-images)
    ("n" (lambda ()(plist-get docker-status-strings :networks))   docker-networks)
-   ("v" (lambda ()(plist-get docker-status-strings :volumes))    docker-volumes)]
+   ("v" (lambda ()(plist-get docker-status-strings :volumes))    docker-volumes)
+   ("x" (lambda ()(plist-get docker-status-strings :contexts)) docker-contexts)]
```

Comment:
Please vertically align `docker-contexts` with the rest :wink:

Reply:
Done!